providers/fips/self_test_data.inc

```diff
@@ -494,8 +494,76 @@ static const ST_KAT_PARAM kbkdf_params[] = {
     ST_KAT_PARAM_END()
 };
 
+#ifndef OPENSSL_NO_TLS1_3
+static const char tls13_kdf_digest[] = "SHA256";
+static int tls13_kdf_extract_mode = 1;
+static int tls13_kdf_expand_mode = 2;
+static const unsigned char tls13_kdf_prefix[] = {
+    0x74, 0x6C, 0x73, 0x31, 0x33, 0x20          /* "tls13 " */
+};
+static const unsigned char tls13_kdf_client_early_secret_label[] = {
```

Comment:
Did these values come from ACVP test data?

Reply:
Mostly.  All values except the intermediate early secret are.  The intermediate value (`tls13_kdf_early_secret`) wasn't specified but it is the output of the first test and the input to the second, so everything is okay end to end.

Reply:
A comment stating where they came from would be useful

Reply:
I've added a comment.

I'll note that none of the other self test KAT data includes a source reference.